lcls_naming_tool/tests/test_cli.py

```diff
@@ -0,0 +1,52 @@
+# Unit tests for the LCLS Naming Tool script
+
+import pytest
+import json
+import os
+from pathlib import Path
+from lcls_naming_tool.lcls_naming_tool import display_version, load_taxons, functional_component_is_valid, fungible_is_valid, constituent_component_is_valid, increment_is_valid, starts_alphanumeric, validate
+
+
+def test_display_version(capsys):
```

Comment:
This test does not pass

Reply:
Hmm, is it the method of testing that doesn't pass or is pytest returning an error? When I run `pytest -v` I get a `test_cli.py::test_display_version PASSED` message.

Reply:
you can see the  test logs under the ctions tab of this repo 
https://github.com/pcdshub/lcls_naming_tool/actions/runs/11075882558/job/30778257851#step:17:85

looks like argparse is pulling args from the pytest invocation.  I think that if you want to use sys.atgs in pytest you need to scrub the old args and provide new ones